src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -112,6 +112,18 @@ namespace ripple {
 
 //------------------------------------------------------------------------------
 
+/** Has the specified time passed?
+
+    @param now  the current time
+    @param mark the cutoff point
+    @return true if \a now refers to a time strictly after \a mark, else false.
+*/
+static inline bool
+after(NetClock::time_point now, std::uint32_t mark)
```

Comment:
This should probably now be a util function since it's used in two files - perhaps in `View.h`, since that file contains a lot of other helper functions.